common/next/next.config.js

```diff
@@ -46,6 +46,12 @@ module.exports = function(webpack) {
     },
   });
 
+  const apmConfig = {
+    environment: process.env.APM_ENVIRONMENT,
+    serverUrl: process.env.APM_SERVER_URL,
```

Comment:
Is this odd that we make this private in the TF and public here?

Reply:
Yeah, perhaps a little - of course it has to be public here because it's needed in the frontend, but generally it's stored alongside the rest of the "secrets" (some of which are not even slightly secret) here https://github.com/wellcomecollection/platform-infrastructure/pull/231